opds.py

```diff
@@ -783,8 +790,13 @@ def from_query(cls, query, _db, feed_name, url, pagination, url_fn, annotator):
         creating an OPDS feed for a custom list and not cached.
 
         TODO: This is used by the circulation manager admin interface.
-        Investigating replacing the code that uses this so that it uses
-        the search index.
+        Investigate changing the code that uses this to use the search
+        index -- this is inefficient and creates an alternate code path
+        that may harbor bugs.
+
+        TODO: This cannot currently return OPDSFeedResponse because the
```

Comment:
What page in the admin is the feed generated from this method used?

Reply:
It looks like it's just CustomListsController.custom_list. The search engine can find all the books on a custom list with no problem. The issue might be that when you're adding books to a custom list, you don't have time to wait around for the search engine to be updated; you need the most up-to-date version of the list.